### ISSUE TITLE
menu: unit tests about keyboard navigation should be added

### ISSUE THREAD

**Issue by DavideMininni-Fincons** [sbb-design-systems/lyne-components#3338]

**Comment by DavideMininni-Fincons** [comment]
@kyubisation @jeripeierSBB 

A question about a possible improvement: 
in other components we have logic also for Home/End/PageUp/PageDown, while in the menu we have it only for arrows.

Should I implement it?

**Comment by kyubisation** [comment]
Sounds reasonable. If it is not very complicated, it would be nice to have. 👍
